src/Components/Toast.jsx

```diff
@@ -26,51 +27,71 @@ function Toast({ message, handleMessageRemove }) {
       clearTimeout(timeoutIdForRemoval);
       clearTimeout(timeoutIdForAnimation);
     };
-  }, []);
+  }, [handleMessageRemove, message.id]);
 
   const styles = {};
 
+  const handleCloseBtn = useCallback(() => {
+    setSlideOut(true);
+    setTimeout(() => {
+      handleMessageRemove(message.id);
+    }, 450);
+  }, [message.id, handleMessageRemove]);
+
   const toastIcon = () => {
-    if (message.toastType === "notice") {
-      styles.backgroundColor = "#D69AFF";
-      return <FiAlertOctagon className="toast-icon" />;
-    } else if (message.toastType === "warning") {
-      styles.backgroundColor = "#CECE4A";
-      return <FiTool className="toast-icon" />;
-    } else if (message.toastType === "success") {
-      styles.backgroundColor = "#5FD65F";
-      return <FiCheckCircle className="toast-icon" />;
-    } else if (message.toastType === "error") {
-      styles.backgroundColor = "#BB4D4D";
-      return <FiAlertTriangle className="toast-icon" />;
-    }
-    return;
+    const iconMap = {
+      notice: {
```

Comment:
@Meet-Kalani Rename it to `info`

Reply:
@Meet-Kalani The change I requested was different

You should supposed to rename `notice`->`info`.